coding-the-legislation/bootstrap_a_new_country.md

```diff
@@ -1,11 +1,7 @@
 # Bootstrap a new country
```

Comment:
My view is that we should always say "tax and benefit system".

Country is, for me, a shortcut in oral language, which adds nothing more. It's a one-to-one relationship.

But I reckon that the term "country" lowers the entry barrier to a newcomer.